lib/moonshot/ssh_fork_executor.rb

```diff
@@ -3,18 +3,18 @@
 module Moonshot
   # Run an SSH command via fork/exec.
   class SSHForkExecutor
-    Result = Struct.new(:output, :exitstatus)
+    Result = Struct.new(:output, :error, :exitstatus)
 
     def run(cmd)
-      output = StringIO.new
-
+      output = error = StringIO.new
       exit_status = nil
-      Open3.popen3(cmd) do |_, stdout, _, wt|
+      Open3.popen3(cmd) do |_, stdout, stderr, wt|
+        error << stderr.read
```

Comment:
Does this need to be `error << stderr.read until stderr.eof?` similar to stdout below?

Reply:
Thanks for the tip!